src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,49 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+  }
```

Comment:
The constructor of the Animal class should add new instances to the static Animal.alive array. This is not currently implemented. To fix this, you should push `this` to the Animal.alive array within the constructor.